src/components/clearBody.js

```diff
@@ -0,0 +1,9 @@
+export const clearBody = () => {
```

Comment:
@Mmarvinn This is not a component, I think it a utility

Reply:
fix